color.go

```diff
@@ -44,6 +45,42 @@ type Attribute int
 
 const escape = "\x1b"
 
+// colorMap contains rich text identifier for color based formatting
+var colorMap = map[string]Attribute{
```

Comment:
I am using this on line 552 in `wrapRichText` to dynamically assign the corresponding color to the symbol matched